src/store/reducers/storage.js

```diff
@@ -243,13 +243,11 @@ export const getFlatListStorageGroups = createSelector([getStoragePools], (stora
                     const limitSizeBytes = _.reduce(
                         group.VDisks,
                         (acc, vDisk) => {
-                            return (
-                                acc +
-                                (Number(vDisk.AvailableSize) ||
-                                    Number(vDisk.PDisk?.AvailableSize) ||
-                                    0) +
-                                (Number(vDisk.AllocatedSize) || 0)
-                            );
+                            const {AvailableSize, AllocatedSize, PDisk} = vDisk;
+                            const available = AvailableSize || PDisk?.AvailableSize || 0;
+                            const allocated = AllocatedSize || 0;
+
+                            return acc + Number(available) + Number(allocated);
```

Comment:
If `AvailableSize` was `'0'`, `PDIsks.AvailableSize` was used, that is incorrect. Now we firstly check if it is defined (it is of `string` type), and only after convert it to `Number`